roles/autotested/templates/control.ini.j2

```diff
@@ -20,11 +20,6 @@ exclude = example,subexample,pretest_example,intratest_example,
 {% if is_atomic %}
           # Atomic host does not have git
           docker_cli/build/git_path,
-
-    {%- if not is_enterprise %}
-          # Only applicable to RHEL Atomic
-          redhat/packaging,
-    {%- endif %}
```

Comment:
584cb45 perhaps? Test was failing on Atomic because of missing `magic.py`, this commit fixed it.